### PR TITLE
feat: rebased to latest ganache-cli version

### PR DESCRIPTION
**Pull request by benjamistan**
Rebased to latest Ganache-CLI
Updated README to new branding.

### CODE REVIEW

**Review by v-stickykeys — COMMENTED** [review]
This is still 72 commits behind trufflesuite:develop
Or is this rebased off of master? In that case should not be merged into develop.

(Also you can add someone as a reviewer in future so we are notified to look at this.)

File: package.json [hunk 1]
```diff
@@ -1,41 +1,76 @@
...
-    "shebang-loader": "0.0.1",
-    "webpack": "^4.18.0",
-    "webpack-cli": "3.1.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/trufflesuite/ganache-cli"
-  },
-  "license": "MIT"
+	"name": "ganache-cli",
```
Comment (v-stickykeys) [inline package.json:2]: Let's keep indentation at 2 spaces here (though 4 is common for json)

**Review by v-stickykeys — COMMENTED** [review]
Just needs commits to fix the conflicts.

For package-lock.json the easiest way to resolve the conflicts is just delete the file, do npm install again and then commit the changes to it.

File: package.json (hunk 1 above)
Comment (benjamistan) [reply package.json:2]: Not sure what's happening - tab indents are at 2 on my machine.

**Pull request merged**